docs/admin/troubleshooting/cluster-health-check.md

```diff
@@ -11,18 +85,23 @@ Log in to `<company-name>.run.ai/dashboards/now`.
 * Go to __Projects__ and create a new Project. Find the new Project using the CLI command: `runai list projects`
 
 
-## Verify that the Run:ai services are running
+### Verify that the Run:ai services are running
 
 Run:
+```
+kubectl get runaiconfig -n runai runai -ojson | jq -r '.status.conditions | map(select(.type == "Available"))'
+```
+Verify that all the Run:ai services are available and have all their required resources available as well.
+
 
+Run:
 ```
 kubectl get pods -n runai
 kubectl get pods -n monitoring
 ```
 Verify that all pods are in `Running` status and a ready state (1/1 or similar)
 
 Run:
-
```

Comment:
@gal-revach markdown needs a space between start and end of code blocks. Do you want to make the changes or should I?

Reply:
@jasonnovichRunAI OK, in the preview it looked OK. If you can please add it, it would be great